dashboard/templates/dashboard.ejs

```diff
@@ -1,79 +1,53 @@
 <%- include('blocks/header', {bot, user, path}) %>
-<div class="container"><h2>Dashboard</h2></div>
-<div class="d-flex justify-content-center">
-  <div class="card-columns"> 
-    <ul class="list-unstyled">
-      <% 
-      let guilds = [];
-      user.guilds.forEach(guild => { 
-        const permsOnGuild = new perms(guild.permissions);
-        if(!permsOnGuild.has("MANAGE_GUILD")) return;
-        if(bot.guilds.cache.get(guild.id)) guilds.unshift(guild);
-        else guilds.push(guild);
-      });
-      guilds.forEach(guild => { 
-        let href, text;
-        let enabled;
-        if(bot.guilds.cache.get(guild.id)) {
-          href = `/dashboard/${guild.id}/manage`;
-          text = "Manage";
-          enabled = true;
-        } else {
-          text = "Invite Bot";
-          enabled = bot.application.botPublic;
-        }
-      %>
-      <li>
-        <div class="card text-white bg-primary mb-3" style="width: 200px;" >
-        <% if(text == "Manage") { %>
-          <div class="img">
-            <% if (guild.icon) { %>
-              <img class="d-block img-thumbnail " style="width: 100%; display: block;" src= "https://cdn.discordapp.com/icons/<%=guild.id%>/<%=guild.icon%>.png?size=128" alt="Card image">
-            <% } else { %>
-              <canvas class="d-block img-thumbnail" style="background-color: #7289da; width: 100%; display: block;" id="<%= guild.name.split(' ').join('').replace(`'`, ``) %>" width="128px" height="128px"></canvas>
-            <script>
-              function draw() {
-                const ctx = document.getElementById("<%= guild.name.split(' ').join('').replace(`'`, ``) %>").getContext('2d');
-                ctx.font = '50px Arial';
-                ctx.textAlign = "center";
-                ctx.fillStyle = "white";
-                ctx.fillText("<%= guild.name.split(' ').map(v => v[0]).join('') %>", 60, 80);
-              };
-              draw();
-            </script>
-            <% } %>
-          </div>
-          <div class="card-footer bg-primary">
-            <h6><%= guild.name %></h6>
-            <a href="<%= href %>"> <%= text %> <i class="fa fa-fw fa-chevron-circle-right" aria-hidden="true"></i></a>
-          </div>
+<h1 class="display-3">Dashboard</h1>
```

Comment:
I can see what you've done, and I do like it, however please align the servers, otherwise it will look cluttered and unorganised.

Also, the MANAGE link does not work, it keeps taking me to the home page.

Reply:
This is the error that is thrown when I try to access the server manage page.

```
TypeError: C:\Users\YorkAARGH\Desktop\Dashboards\New\dashboard\templates\guild\manage.ejs:8
    6|   <div class="row" style="margin-top: 25px;">
    7|     <div class="col">
 >> 8|       <%- include('../blocks/guild-card', {guild}) %>
    9|     </div>
    10|     <div class="col">
    11|       <form method="post">

C:\Users\YorkAARGH\Desktop\Dashboards\New\dashboard\templates\blocks\guild-card.ejs:24
    22|       </li>
    23|       <li class="list-group-item d-flex justify-content-between align-items-center" style="width: 100%" data-toggle="tooltip" title="Server Owner.">
 >> 24|         <i class="fa fa-fw fa-user-circle-o" aria-hidden="true"></i>Owner: <span id="owner"><%= `${guild.owner.user.tag}` %></span>
    25|       </li>
    26|         <li class="list-group-item d-flex justify-content-between align-items-center" data-toggle="tooltip" title="Members">
    27|           <i class="fa fa-fw fa-users" aria-hidden="true"></i>Members <span class="badge badge-primary badge-pill"><%= `${guild.memberCount}` %></span>

Cannot read property 'user' of null
    at eval (eval at compile (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:662:12), <anonymous>:40:41)
    at guild-card (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:692:17)
    at include (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:690:39)
    at eval (eval at compile (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:662:12), <anonymous>:14:17)
    at manage (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:692:17)
    at tryHandleCache (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:272:36)
    at View.exports.renderFile [as engine] (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\ejs\lib\ejs.js:489:10)
    at View.render (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\express\lib\view.js:135:8)
    at tryRender (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\express\lib\application.js:640:10)
    at Function.render (C:\Users\YorkAARGH\Desktop\Dashboards\New\node_modules\express\lib\application.js:592:3)
TypeError: C:\Users\YorkAARGH\Desktop\Dashboards\New\dashboard\templates\guild\manage.ejs:8
    6|   <div class="row" style="margin-top: 25px;">
    7|     <div class="col">
 >> 8|       <%- include('../blocks/guild-card', {guild}) %>
    9|     </div>
    10|     <div class="col">
    11|       <form method="post">
```

Reply:
Ok, i will check

Reply:
its because of `guild.owner.user.tag` and it can be fixed with Intents for cache